packages/react-native/ReactCommon/react/renderer/graphics/ValueUnit.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+namespace facebook::react {
+
+enum class UnitType {
+  UNDEFINED,
+  POINT,
+  PERCENT
+};
+
+struct ValueUnit {
+  float value;
+  UnitType unit;
+  
+  ValueUnit() : value(0.0f), unit(UnitType::UNDEFINED) {}
```

Comment:
nit: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-in-class-member-initializers-instead

```suggestion
  float value{0.0f};
  UnitType unit{UnitType::Undefined};
```